catbot:
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 50

    # Effort Controllers
    front_left_wheel_effort_controller:
        type: effort_controllers/JointEffortController
        joint: front_left_wheel_joint
        pid: {p: 100.0, i: 0.01, d: 10.0}
    front_right_wheel_effort_controller:
        type: effort_controllers/JointEffortController
        joint: front_right_wheel_joint
        pid: {p: 100.0, i: 0.01, d: 10.0}
    back_left_wheel_effort_controller:
        type: effort_controllers/JointEffortController
        joint: back_left_wheel_joint
        pid: {p: 100.0, i: 0.01, d: 10.0}
    back_right_wheel_effort_controller:
        type: effort_controllers/JointEffortController
        joint: back_right_wheel_joint
        pid: {p: 100.0, i: 0.01, d: 10.0}
